ckanext/dcat/plugins/pylons_plugin.py

```diff
@@ -32,7 +32,7 @@ def before_map(self, _map):
                          action='read_catalog')
 
             _map.connect('add dataset', '/dataset/new', controller='package', action='new')
-            _map.connect('dataset_read', '/dataset/{_id}',
+            _map.connect('dataset.read', '/dataset/{_id}',
```

Comment:
I'm not sure Pylons will understand `dataset.read`, this is probably what is causing the 2.7/2.8 failures